test/test_roo.rb

```diff
@@ -282,6 +282,12 @@ def test_cell_multiline
     end
   end
 
+  def test_apostrophe_replacement
+    with_each_spreadsheet(:name=>'apostrophe', :format=>[:openoffice]) do |oo|
```

Comment:
Please add apostrophe to `test/files/apostrophe.ods`

Reply:
Hey @kakubin
please have another look at the commit.
The file is exactly here: `test/files/apostrophe.ods`

Are you having trouble running the tests?

Reply:
I opened the file in local and it didn't contain an apostrophe.

Reply:
Yes this is correct. The issue itself is not really due to having an apostrophe in the file, but the `gsub!` method that is forcing to replace one just in case.

It can be reproduced with an .ods file that has an empty string `''` which is the most minimal step to reproduce the issue with the frozen string literal.

Reply:
@kakubin I am also open to rename this test to something along the lines of "html escape" or "frozen string" or "gsub replacement" if this might be better suitable for this edge case.

WDYT ?

Reply:
I would recommend renaming this test case and the file to `frozen_string_gsub_replacement` to make it more clear that this is testing a very specific case. 

There should be a separate test for the `&apos; > '` replacement, I agree. Problem is once I save it with my version of open office than for some reason I won't be able to reproduce the actual frozen string issue anymore.

Therefor I would:
1. Rename this test case to make it's existence more clear
2. Create a separate test case to just these the apostrophe replacement

WDYT ?